File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>
void spaces(int n)
{
    for (int i = 0; i < n; i++){
        printf("%c", ' ');
    }
}
void hashes(int n)
{
    for (int i = 0; i < n; i++){
        printf("%c", '#');
    }
}
int main(void)
{
    int ht = 0;
    do{
        ht = get_int("Height:");
    }while ((ht < 1) || (ht > 8));


    for (int i = 0; i < ht; i++)
    {
        spaces(ht-(i+1));
        hashes(i+1);
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

def print_spaces(n):
    print(" "*n, end="")

def print_hashes(n):
    print("#"*n, end="")


def main():
    ht = 0
    while ht < 1 or ht > 8:
        ht = get_int("Height: ")

    for i in range(1, ht+1):
        print_spaces(ht-i)
        print_hashes(i)
        print_spaces(2)
        print_hashes(i)
        print()
main()



